languoids/tree/grea1241/md.ini

```diff
@@ -6,9 +6,10 @@ level = family
 [classification]
 sub = **hh:hv:Manoharan:Andamanese**
 familyrefs = 
-	**hh:hv:Abbi:Andamanese**
+	**hh:hv:Abbi:Andamanese** **hh:typ:ComrieZamponi:Great-Andamanese**
```

Comment:
Ok it's the result of l.cfg.set('classification', 'familyrefs', ["**hh:hv:Abbi:Andamanese**", "**hh:typ:ComrieZamponi:Great-Andamanese**"]), should if be done elsehow or maybe I have and unupdated pyglottolog?!